client/src/components/ManageBookings/BookModule/EachBooking/EachBooking.tsx

```diff
@@ -0,0 +1,52 @@
+import { Grid, CircularProgress, Typography, Avatar, Button } from '@material-ui/core';
+import useStyles from './useStyles';
+import { Request } from '../../../../interface/Request';
+
+interface Props {
+  className?: string;
+  request: Request;
+  offer?: boolean;
+  handleAccept?: (arg1: boolean, arg2: number) => void;
+  id: number;
+}
+
+const EachBooking = ({ className, request, offer, handleAccept, id }: Props): JSX.Element => {
+  const classes = useStyles();
+  const month = request.date.toLocaleString('default', { month: 'long' });
+  const stringDate = `${request.date.getDate()} ${month} ${request.date.getFullYear()}`;
```

Comment:
This works, do you think using [DateTime Format](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat/DateTimeFormat) be better here?

Reply:
Interesting! I wasn't aware of this option. Since I'm also doing the integration I'll try to change it there once I've had a chance to check the docs.